es/index.js

```diff
@@ -137,12 +137,12 @@ function (_React$Component) {
       if (this.props.autoplay) {
         this.startAutoplay();
       }
-    } // @TODO Remove deprecated componentWillReceiveProps with getDerivedStateFromProps
+    } // @TODO Remove deprecated UNSAFE_componentWillReceiveProps with getDerivedStateFromProps
     // eslint-disable-next-line react/no-deprecated
 
   }, {
-    key: "componentWillReceiveProps",
-    value: function componentWillReceiveProps(nextProps) {
+    key: "UNSAFE_componentWillReceiveProps",
+    value: function UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
So... you actually don't want to change this file at all. This is a generated file that is packagaed up when nuka is released. You want to add this change to `src/index.js` 👍

Reply:
@cdfunderburk https://github.com/shipt/nuka-carousel/blob/master/src/index.js#L113

Reply:
Ah sorry we were talking about the index.js and I was still focused on the orginal file we looked at.  Will fix this